packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -86,94 +212,110 @@ def create_client(
     """
     channel_pool = GrpcChannelPool()
     discovery_client = DiscoveryClient(grpc_channel_pool=channel_pool)
-    built_in_import_modules: List[str] = []
-    custom_import_modules: List[str] = []
 
     if all:
```

Comment:
This function is getting big, so please extract each case into a helper function:
- `_create_all_clients`
- `_create_clients_interactively`
- `_create_clients`

Reply:
Moved to helper methods.